{"status": "resolved", "priority": "urgent", "title": "Some profiler improter crashes", "milestone": "0.11.02", "superceder": "", "nosylist": ["paulproteus"], "assigned": "paulproteus", "waitingon": "", "keywords": [], "id": "296", "files": [], "history": [{"message": "I think these are handled, and I should receive exception emails now if they're\nnot. I'll open separate tickets for those; marking this as resolved.\n   \n", "author": "paulproteus"}, {"message": "This might be preventing DIA processing.\n\n  File \"/home/deploy/milestone-a.buildout/mysite/customs/profile_importers.py\",\nline 122, in __call__     value = self.fn(*args, **kwargs)   File\n\"/home/deploy/milestone-a.buildout/mysite/customs/profile_importers.py\", line\n59, in squashIrrelevantErrors     raise error.value\n   \n", "author": "paulproteus"}]}